app/src/main/java/org/piramalswasthya/cho/repositories/DeliveryOutcomeRepo.kt

```
@@ -64,4 +69,286 @@ class DeliveryOutcomeRepo @Inject constructor(
+                return false
+            }
+
+            val responseString = response.body()?.string()
+            if (responseString.isNullOrBlank()) {
+                Timber.d("DeliveryOutcome saveAll succeeded with empty response body")
+                return true
+            }
+
+            val jsonObj = try {
+                JSONObject(responseString)
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)